cheatsheets/gleam-for-python-users.md

```diff
@@ -47,9 +47,8 @@ In Python, comments are written with a `#` prefix.
 A docstring (matching """) that occurs as the first statement in a module, function, class, or method definition will become the `__doc__` attribute of that object.
 
 ````py
-def a_method():
-    """some documentation for this method"""
-    pass
+def a_function():
+    """Return some important data."""
```

Comment:
Why is it that the `pass`s have been removed from the functions? To me the code looks incomplete now.

Reply:
The reason I removed them is, that they aren't needed to be valid Python code. But from a documentation standpoint it's probably better to leave them in or use an ellipsis. If you want I can add them back, no problem.

Reply:
Yes please! Just to make that extra bit clear for people bad at Python like myself :)

Reply:
Added it back. :)